cmd/axelard/cmd/vald/start.go

```diff
@@ -195,7 +195,7 @@ func listen(ctx sdkClient.Context, txf tx.Factory, axelarCfg config.ValdConfig,
 	eventBus := createEventBus(tmClient, startBlock, logger)
 
 	tssMgr := createTSSMgr(bc, ctx, axelarCfg, logger, valAddr, cdc)
-	if recoveryJSON != nil && len(recoveryJSON) > 0 {
```

Comment:
why remove the nil check?

Reply:
Eh, VS code linter was complaining since `len(nil) = 0` array in Go. I can switch it back if you prefer.

Reply:
I am good with it, just curious